cmd/hub/mcscontroller/main.go

```diff
@@ -63,7 +67,8 @@ func main() {
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
-	flag.StringVar(&namespace, "ns", corev1.NamespaceAll, "namespace to watch, defaults to NAMESPACE environment")
+	flag.StringVar(&namespace, "ns", os.Getenv(ENV_NAMESPACE),
```

Comment:
if no env var and no `-ns` you will get `nil` right ? do you handle this correctly?

Reply:
why not having a default value for ns: submariner-k8s-broker